spec/models/cohort_analytics_spec.rb

```diff
@@ -27,6 +27,10 @@
     end
   end
 
+  before(:all) do
+    GrdaWarehouse::Cohorts::CohortColumn.maintain!
+  end
+
```

Comment:
Maybe this should go in `rails_helper.rb`.  There's a `before(:suite)` section that probably makes sense.